bin/start

```diff
@@ -37,11 +39,13 @@ RUN_IN_DOCKER=false
 DEMO=false
 CONJUR_DEPLOYMENT=oss
 SUMMON_ENV=gke
+ENV_CONFIG=test
 
 while true ; do
   case "$1" in
     --docker ) RUN_IN_DOCKER=true ; shift ;;
     --demo ) DEMO=true ; shift ;;
+    --dev ) ENV_CONFIG=dev ; shift ;;
```

Comment:
will `--dev` and `--docker` work together? i remember we had an issue to run docker locally